generated/user_existence/user_existence.json

```diff
@@ -0,0 +1,29 @@
+{
+  "primary_applicant": {
+    "id": "58bf3bb6e41b0738bf0000ce",
+    "user_id": "58bf3b87e41b0738bf0000cc",
+    "first_name": "Loren",
+    "last_name": "Dimallo"
```

Comment:
hmm, so here's the problem with this. they're not logged in at this point; this is an anonymous API. If we expose an anonymous "give me a SSN and I'll tell you people's first & last name and employers" API, we've built a PII-harvesting machine. There are only a billion possible SSNs, so it shouldn't take very long to harvest all the DC users' private info this way. So that's clearly not going to work, and we have to rethink this. We have to verify that they're who they say they are before we tell them anything, and I think the easiest way to do that is to piggyback this onto the RIDP answer API. Sorry I didn't think of this earlier -- I only really understood it when I saw this code!